doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -119,7 +119,9 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
-|no_changes_yet_note|
+* :ref:`ble_nrf_dm` sample:
+
+  * Align the configuration to the Distance Measurement module.
```

Comment:
This changelog entry does not really tell me anything.
Maybe something like:
- Split the configuration of the Distance Measurement submodule from the Nordic Distance Measurement library. This allows to use the Nordic Distance Measurement library without the Distance Measurement submodule.

Reply:
Good point. Fixed.